certbot/docs/install.rst

```diff
@@ -6,80 +6,49 @@ Get Certbot
    :local:
 
 
-About Certbot
-=============
 
-*Certbot is meant to be run directly on a web server*, normally by a system administrator. In most cases, running Certbot on your personal computer is not a useful option. The instructions below relate to installing and running Certbot on a server.
+.. _system_requirements:
 
-System administrators can use Certbot directly to request certificates; they should *not* allow unprivileged users to run arbitrary Certbot commands as ``root``, because Certbot allows its user to specify arbitrary file locations and run arbitrary scripts.
+System Requirements
+===================
 
-Certbot is packaged for many common operating systems and web servers. Check whether
-``certbot`` (or ``letsencrypt``) is packaged for your web server's OS by visiting
-certbot.eff.org_, where you will also find the correct installation instructions for
-your system.
+- Python 3.7+
+- UNIX-like operating system
+- Root access
+- Port 80 Open
```

Comment:
On the fence about this one. I guess it's true for the majority of users, if we go by the authenticator statistics that show HTTP is the most common challenge method.

On the other hand, it's not an installation requirement. Maybe this belongs in the User Guide, where we describe the plugin choices.

Reply:
For reference, I tried to model this section after the Certbot home page where the list was simple.
![Screenshot 2022-06-02 at 11-39-14 Certbot](https://user-images.githubusercontent.com/4276080/171702671-efc51e5c-3855-4ba9-86ae-b7f3b3d37099.png)